client/src/components/page-profile.component.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { AppDispatch, RootState } from '../app/store';
+import { getUserData } from '../features/user/userActions';
```

Comment:
Weird file structure - better to have like `module`, in it - `actions`, `reducer`, `index`(main React controller), `types` etc - `import { getUserData } from '../modules or containers/user/actions';`